web/app/public/index.html

```diff
@@ -11,7 +11,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
     <link rel="shortcut icon" href="/favicon.ico" />
-    <meta property="description" content="Gatus is an advanced automated status page that lets you monitor your applications and configure alerts to notify you if there's an issue" />
+    <meta property="description" content="{{ .Description }}" />
```

Comment:
You also have to regenerate the static file by executing `make frontend-build`.

That said, if it's easier for you, I can take care of it myself :)

Reply:
I don't mind — it would take me a few hours before I have the opportunity (provided I remember!), so if you could do it, that'd be great! :)